### PR TITLE
modified the decending/ascending symbol of issue#5802

### LINKED ISSUE #5802: Wrong symbol for ascending/descending sorting in card browser
###### Reproduction Steps

1. Open card browser
2. Menu > change display order
3. Look at the symbol after the current selection

###### Expected Result

Ascending symbol for ascending sorting 

###### Actual Result

Ascending symbol for descending sorting. See screenshot: the sort field is sorted Z to A (descending) but the triangle looks upwards/ascending.

Screenshot_20200310-094008~(1)

###### Debug info
AnkiDroid Version = 2.10alpha45

Android Version = 9

ACRA UUID = 1f29f610-893a-403f-83d5-25a26ba97fe0

###### Research
*Enter an [ x ] character to confirm the points below:*

[X  ] I have read the [support page](https://ankidroid.org/docs/help.html) and am reporting a bug or enhancement request specific to AnkiDroid

[ x ] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue

[  x] I have searched for similar existing issues here and on the user forum

### PR DESCRIPTION
**Pull request by jianersswlq**
## Purpose / Description
Wrong symbol for ascending/descending sorting in card browser.

## Fixes
Fixes #5802.

## Approach
Reverse the logic of the symbol in *CardBrowserOrderDialog.java*.

## How Has This Been Tested?
Reproduction Steps:
+ Open card browser
+ Menu > change display order
+ Look at the symbol after the current selection

With Android Version = 9

## Checklist
_Please, go through these checks before submitting the PR._

- [ ] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [ ] You have a descriptive commit message with a short title (first line, max 50 chars).
- [ ] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [ ] You have commented your code, particularly in hard-to-understand areas
- [ ] You have performed a self-review of your own code

### CODE REVIEW

**Comment by mikehardy** [comment]
@RSBat you looked into #5802 - what do you think of this?

**Comment by RSBat** [comment]
Arrow directions are more consistent with desktop but sorting "by sort field" now has wrong direction by default. Removing [these lines](https://github.com/ankidroid/Anki-Android/blob/dcfd5bd27e88b999f76de18e322776cadb40ef6d/AnkiDroid/src/main/java/com/ichi2/anki/CardBrowser.java#L513-L515) should do the trick. I thought it would break something on update, but can't remember why.

Here are some steps that are broken now but will get fixed by the above:
1. Open card browser. Sorting by sort field must be selected (if not - select it and reopen). Check if direction matches order
2. Choose another order
3. Choose "by sort field". Check if direction matches
One of them will be ok, another one won't match.

And another case:

1. Open card browser. Sorting by sort field must be selected (if not - select it and reopen)
2. Change direction twice - it will be same as when opened
3. Reopen card browser - direction will be different
4. Repeat these steps to get original direction again

**Comment by mikehardy** [comment]
So @RSBat if I understand correctly - this PR should be modified to also delete the small code block you linked to, and then the combination of the current PR plus the deletion of those lines seems right to you?

**Comment by RSBat** [comment]
Yes @mikehardy

File: AnkiDroid/src/main/java/com/ichi2/anki/AbstractFlashcardViewer.java [hunk 1]
```diff
@@ -130,6 +132,7 @@
 
 import static com.ichi2.anki.cardviewer.CardAppearance.calculateDynamicFontSize;
 import static com.ichi2.anki.cardviewer.ViewerCommand.*;
+import static com.ichi2.anki.reviewer.CardMarker.*;
```
Comment (timrae) [inline AnkiDroid/src/main/java/com/ichi2/anki/AbstractFlashcardViewer.java:135]: Codacy Issue found: [Avoid unused imports such as 'com.ichi2.anki.reviewer.CardMarker'](https://app.codacy.com/manual/timrae/Anki-Android/pullRequest?prid=5490756)

**Comment by timrae** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- AnkiDroid/src/main/java/com/ichi2/anki/AbstractFlashcardViewer.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/timrae/Anki-Android/pullRequest?prid=5490756&bid=17993539)

**Comment by mikehardy** [comment]
Superceded by #6326

**Pull request closed without merging**